weathergov/frontend/assets/sass/components/daily.scss

```
@@ -74,3 +76,46 @@ wx-daily-forecast {
 .wx-daily-forecast-list-item-inner[role="tabpanel"][data-tabpanel-active="true"] {
     display: revert;
 }
+
+/* Styles for the daily forecast list item
+--------------------------------------------------- */
+.wx-daily-forecast-item-summary-area--other {
+    margin-bottom: units(1);
+
+    > span {
+        @extend .wx-marker-before--indigo;
+    }
+}
+
+.wx-daily-forecast-item-summary-area--overnight {
+    margin-bottom: units(1);
+
+    > span {
+        @extend .wx-marker-before--midnight-indigo;
+    }
+}
+
+.wx-daily-forecast-item-summary-area--daytime {
+    margin-bottom: units(1);
```

Need extra space on mobile
```suggestion
    margin-bottom: units(3);
    
    @include at-media(tablet) {
        margin-bottom: units(1);
    }
```